editor/import/dynamic_font_import_settings.cpp

```diff
@@ -1345,6 +1345,7 @@ DynamicFontImportSettingsDialog::DynamicFontImportSettingsDialog() {
 	page2_description->set_text(TTR("Add font size, and variation coordinates, and select glyphs to pre-render:"));
 	page2_description->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	page2_description->set_autowrap_mode(TextServer::AUTOWRAP_WORD_SMART);
+	page2_description->set_size(Size2(500, 1) * EDSCALE);
```

Comment:
I don't see any fixed size for this dialog, so it's worth to double-check if 500 is not too much, This value should be rather conservative (applies to all 3 labels).